mep/settings.py

```diff
@@ -295,10 +293,10 @@
 f = os.path.join(BASE_DIR, "mep", "local_settings.py")
 if os.path.exists(f):
     import sys
-    import imp
+    import types
 
     module_name = "mep.local_settings"
-    module = imp.new_module(module_name)
+    module = types.ModuleType(module_name)
     module.__file__ = f
     sys.modules[module_name] = module
     exec(open(f, "rb").read())
```

Comment:
Ooh, I forgot that s&co was still using this style import for local settings. Thank you for finding a solution.

Could you estimate level of effort for migrating to django-split-settings?

Reply:
Seems simple enough, I would say low effort.